Tests/test_imagefont.py

```diff
@@ -440,6 +440,21 @@ def test_unicode_pilfont(self):
         with self.assertRaises(UnicodeEncodeError):
             font.getsize(u"’")
 
+    @unittest.skipIf(sys.platform.startswith('win32') and sys.version.startswith('2'),
+                     "requires Python 3.x on Windows")
+    def test_unicode_extended(self):
+        # issue #3777
+        text = u"A\u278A\U0001F12B"
+        target = "Tests/images/unicode_extended.png"
+
+        ttf = ImageFont.truetype("Tests/fonts/NotoSansSymbols-Regular.ttf",
+                                 FONT_SIZE, layout_engine=self.LAYOUT_ENGINE)
+        img = Image.new("RGB", (100, 60))
+        d = ImageDraw.Draw(img)
+        d.text((10, 10), text, font=ttf)
+
+        self.assert_image_similar_tofile(img, target, self.metrics['textsize'])
```

Comment:
Bear with me if this is a silly question, but why is the third argument `self.metrics['textsize']`?

Reply:
I don't know what these values mean, I just guessed based on the other tests. Looks like changing it to `self.metrics['multiline']` would fix the failing builds. Do you happen to know the meaning of these values?

Reply:
Oh, I see what you mean - the other tests do have similar lines.

The function called is https://github.com/python-pillow/Pillow/blob/e8b2da587f721e9e7fee7135dee159b9330098f4/Tests/helper.py#L152

The third argument, epsilon, is a measure of how much different the first two images can be and still allow the test to pass.

Reply:
~~Yes, I found that, but I can't find a reason why these values were chosen and named as they were.~~
Duh, use `git blame`: https://github.com/python-pillow/Pillow/commit/20da797fcee7cfa22ee9637cb8b3b041e1fa8ecb sort-of explains this. I'll just change the value as I wrote earlier.